nncf/experimental/onnx/engine.py

```diff
@@ -34,8 +34,11 @@ def __init__(self, **rt_session_options):
         self._inputs_transforms = lambda input_data: input_data.astype(np.float32)
         self.sess = None
         self.rt_session_options = rt_session_options
-        if 'providers' not in self.rt_session_options:
-            self.rt_session_options['providers'] = ['OpenVINOExecutionProvider']
+
+        # TODO: Do not force it to use CPUExecutionProvider
+        # OpenVINOExecutionProvider raises the following error.
+        # onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : This is an invalid model. Error: Duplicate definition of name (data).
+        self.rt_session_options['providers'] = ['CPUExecutionProvider']
```

Comment:
Is this okay to set `providers` as `CPUExecutionProvider` instead of setting from config file?

Reply:
Temporarily fixed as `CPUExecutionProvider` regardless of the input configuration. This is because an error has been shown if we use `OpenVINOExecutionProvider`. I left a TODO comment so we can look back.